db/migrate/20220226115707_create_products.rb

```diff
@@ -0,0 +1,10 @@
+class CreateProducts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :products do |t|
+      t.string :name
```

Comment:
If you add validation to the model for fields presence, it would be good to also have it on database level. In migrations you can set `null: false`